src/PSRule.Rules.Azure/rules/Azure.CDN.Rule.ps1

```diff
@@ -38,4 +38,22 @@ Rule 'Azure.CDN.MinTLS' -Ref 'AZR-000092' -Type 'Microsoft.Cdn/profiles/endpoint
     }
 }
 
+# Synopsis: Use Azure Front Door Standard or Premium SKU to improve the performance of web pages with dynamic content and overall capabilities.
+Rule 'Azure.CDN.UseFrontDoor' -Ref 'AZR-000286' -Type 'Microsoft.Cdn/profiles' -If { IsStandard_MicrosoftSKU } -Tag @{ release = 'GA'; ruleSet = '2022_09' } {
+    $Assert.In($TargetObject, 'sku.name', @('Standard_AzureFrontDoor', 'Premium_AzureFrontDoor'));
+}
+
 #endregion Rules
+
+#region Helper functions
+
+function global:IsStandard_MicrosoftSKU {
+    [CmdletBinding()]
+    [OutputType([System.Boolean])]
+    param ()
+    process {
+        return $Assert.Contains($TargetObject, 'sku.name', 'Standard_Microsoft');
```

Comment:
We should have passing resources. Let's add the passing cases so that rule runs for those.

```suggestion
        return $Assert.In($TargetObject, 'sku.name', @('Standard_Microsoft', 'Standard_AzureFrontDoor', 'Premium_AzureFrontDoor'));
```

Reply:
I agree!